src/main/dist/conf/sidecar.yaml

```
@@ -75,6 +75,7 @@ sidecar:
   sstable_upload:
     concurrent_upload_limit: 80
     min_free_space_percent: 10
+    # file_permissions: # when not specified, the default permissions are "rw-r--r--"
```

Rephrase the comment so the part before `#` is a valid form, it provides the guidance on the acceptable format of the value. 
```suggestion
    # file_permissions: "rw-r--r--" # when not specified, the default file permissions is owner read & write, others & group read
```